flow/connectors/eventhub/eventhub.go

```diff
@@ -215,24 +226,11 @@ func (c *EventHubConnector) processBatch(
 }
 
 func (c *EventHubConnector) SyncRecords(req *model.SyncRecordsRequest) (*model.SyncResponse, error) {
-	maxParallelism := req.PushParallelism
-	if maxParallelism <= 0 {
-		maxParallelism = 10
-	}
-
 	var err error
 	batch := req.Records
 	var numRecords uint32
 
-	shutdown := utils.HeartbeatRoutine(c.ctx, 10*time.Second, func() string {
-		return fmt.Sprintf(
-			"processed %d records for flow %s",
-			numRecords, req.FlowJobName,
-		)
-	})
-	defer shutdown()
-
-	numRecords, err = c.processBatch(req.FlowJobName, batch, maxParallelism)
+	numRecords, err = c.processBatch(req.FlowJobName, batch)
```

Comment:
```suggestion
	numRecords, err := c.processBatch(req.FlowJobName, batch)
```
then get rid of var lines